### PR TITLE
all migration and association of application

### PR DESCRIPTION
**Pull request by AmiFaldu-BTC**

### CODE REVIEW

File: app/assets/javascripts/chartkick.js [hunk 1]
```diff
@@ -0,0 +1,1429 @@
+/*
```
Comment (PaarthBarot) [inline app/assets/javascripts/chartkick.js]: @VaibhaviJoshi-BTC This file should be a minified version.

File: app/assets/stylesheets/font-awesome.css [hunk 2]
```diff
@@ -0,0 +1,2199 @@
+/*!
```
Comment (PaarthBarot) [inline app/assets/stylesheets/font-awesome.css:1]: @VaibhaviJoshi-BTC Same as above.

File: app/controllers/customer_controller.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+class CustomerController < ApplicationController
+  before_action :set_user, only: [:new, :new_family_member]
+
+  def new
+  end
+
+  def create
+    customer = User.new(customer_params)
+    customer.password = 'aaaaaa'
```
Comment (PaarthBarot) [inline app/controllers/customer_controller.rb]: @VaibhaviJoshi-BTC Hardcoded password?

**Review by PaarthBarot — CHANGES_REQUESTED** [review]
please fix the comments.

File: db/migrate/20160922090029_create_personal_infos.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreatePersonalInfos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :personal_infos do |t|
```
Comment (amitpatelx) [inline db/migrate/20160922090029_create_personal_infos.rb:3]: need to add indexes on all fields on which you will search

File: db/migrate/20160926063859_create_categories.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration[5.0]
+  def change
+    create_table :categories do |t|
+      t.string :name
```
Comment (amitpatelx) [inline db/migrate/20160926063859_create_categories.rb]: need to add index

File: db/migrate/20160926064046_create_companies.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+class CreateCompanies < ActiveRecord::Migration[5.0]
+  def change
+    create_table :companies do |t|
+      t.string :name
```
Comment (amitpatelx) [inline db/migrate/20160926064046_create_companies.rb]: need to add indexes

File: db/migrate/20160926073437_create_plans.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+class CreatePlans < ActiveRecord::Migration[5.0]
+  def change
+    create_table :plans do |t|
+      t.string :name
```
Comment (amitpatelx) [inline db/migrate/20160926073437_create_plans.rb]: need to add indexes

File: db/migrate/20160926085454_create_lics.rb [hunk 8]
```diff
@@ -0,0 +1,15 @@
+class CreateLics < ActiveRecord::Migration[5.0]
+  def change
+    create_table :lics do |t|
```
Comment (amitpatelx) [inline db/migrate/20160926085454_create_lics.rb]: Use full name life_insurance

File: db/migrate/20160926090430_create_vehicles.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+class CreateVehicles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :vehicles do |t|
+      t.string :registration_number
```
Comment (amitpatelx) [inline db/migrate/20160926090430_create_vehicles.rb]: need to add index

**Review by amitpatelx — CHANGES_REQUESTED** [review]
Read http://rakeroutes.com/blog/increase-rails-performance-with-database-indexes/ why we need to add indexes

**Pull request merged**